ONSdigital/florence
# added sort class with static function to sort alphabetically

### What

Datasets on the "Select a dataset" page weren't being ordered alphabetically. This PR fixes that by having a new `sort` class in the `utilities` directory. As there is another ticket that requires a list to be ordered alphabetically, it seemed better to create a common function for this logic.

### How to review

- Check the code changes make sense
- Go to the "Select a dataset" page from within a collection in Florence and ensure that the datasets are ordered alphabetically

### Who can review

Anyone but me
